docs/docs/quick-start.md

```diff
@@ -22,13 +22,13 @@ npm install -g gatsby-cli
 ### Create a new site
 
 ```shell
-gatsby new gatsby-site
+gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world 
 ```
 
 ### Change directories into site folder
 
 ```shell
-cd gatsby-site
+cd hello-world
```

Comment:
What's the thinking behind changing this? (and above ☝️ )

Not against it -- just curious!

Reply:
Just figured it would better signal that this was a smaller quick start. You are literally doing a hello-world and not a full site. Not a necessary part of the change!

Reply:
I'm cool with it!

Reply:
We use the Quick Start as a jumping off point for other docs, including the manual recipes section: https://gatsbyjs.org/docs/recipes/ 

It would be good to consider those use cases in addition to a standalone Quick Start exercise. But I'd defer to @AishaBlake!

Reply:
Good point! Best as I can tell, the recipe at `docs/recipes/working-with-plugins.md` would need to be updated.

Reply:
Many of them list "A Gatsby site" as a prerequisite, and they link to the Quick Start. Likely in other parts of the docs, too.

Reply:
I think that's phraseology? I don't see that being confusing after this change?

Reply:
Got it. I think leaving the new working directory name as is is probably fine and not essential to this test. If we find it's successful and that it would be a useful long term change, we update the directory name here and the name in the links in the current manual recipes section.

As best I can tell, none of the recipes are dependent on having the default starter from the Quick Start so this shouldn't affect users going through those.

Reply:
Working directory name was reverted to `gatsby-site` on #https://github.com/gatsbyjs/gatsby/pull/25914/commits/d0c9544cd4cd730acc89c6865792c99964b5002e